api/config/service/shortlink/v1/shortlink.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package clutch.config.service.shortlink.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/config/service/shortlink/v1;shortlinkv1";
+
+message Config {
+}
```

Comment:
Is there a reason to have the config?

Reply:
yeah i can delete it for now, i was debating making some things configurable, like the length of the hash but i think that might be overkill.